src/runtime/storageNG/tests/store-test.ts

```diff
@@ -156,6 +158,7 @@ describe('Store', async () => {
     const count = new CRDTCount();
     count.applyOperation({type: CountOpTypes.Increment, actor: 'me', version: {from: 0, to: 1}});
 
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
remove this line, remove `async` from promise executor and remove `await` before `driver.receiver`

Reply:
done